### DISCUSSION TITLE
Getting error "Controller is unable to transmit"

### DISCUSSION THREAD

**Discussion by iscofield** [zwave-js/zwave-js#6561] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?

Per the last comment in this previous support ticket, I'm opening a new support case here: https://github.com/zwave-js/node-zwave-js/issues/6316.

I am having the exact same issue.  I get the error "Controller is unable to transmit" and I have to do a soft reset to get the network to work, but it stops working again as soon as I ping a node.

I have four light switches that I am trying to add through a zooz 800 series range extender and they won't add.  They just stay stuck in nodeinfo and appear dead.  NWI doesn't seem to work, and the controller seems to be freaking out.

What did you expect to happen instead?
I don't see the error message, and pings work for nodes as expected. 

  2. Click on '...'
  3. Scroll down to '...'

Ping range extender node.  Immediately see "controller is unable to transmit" error.  Ping then fails with no response.
Soft reset of controller.
Ping range extender node and get a success.
Ping second node, and controller then shows error again.

Anything else we should know?

### Software versions

zwave-js-ui: 9.3.2
zwave-js: 12.3.0

Home Assistant Z-Wave Integration: ...

Home Assistant Z-Wave JS Addon: 3.02

ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ... N/A

### Device information

Manufacturer: Zooz
Model name: ZST39LR
Node ID: range extender: 0x027a 0x0004-0x0510
cabana_fan: 0x027a 0x7000-0xa001

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [x] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwave-js-ui-store.zip](https://github.com/zwave-js/node-zwave-js/files/13710515/zwave-js-ui-store.zip)

**Comment by AlCalzone** [comment, marked as answer]
Unfortunately, this is a known issue with the 700/800 series firmware. 7.19.x seems especially bad.
Silabs promised an update in December, but AFAIK the recently released 7.21.0 doesn't contain the fix yet. Hopefully 7.21.1 will fix that.

**Reply by iscofield** [reply]
Thanks for the update!  Do you know where I could subscribe / track to see when a new update is released so I can apply it?  I'm very anxiously awaiting this so I don't have to keep rebooting my controller when my network goes down.

**Reply by kpine** [reply]
https://github.com/SiliconLabs/gecko_sdk/releases/

**Reply by Hedda** [reply]
Still listed as known issue in 7.21.2 so unfortunate that was not the answer. Silabs say that they are still woking on finding a solution for future firmware updates.

**Comment by fpytloun** [comment]
I have similar issue with RaZberry board. Controller gets into unable to transmit state after few days. After performing soft-reset, it works again for some time.

Does anyone know how to call soft-reset from a scheduled job? I was not able to find this in docs.

**Reply by AlCalzone** [reply]
Razberry 7?

**Reply by fpytloun** [reply]
Yes, Razberry 7 Pro. I am facing issues with this device since day one 😞 I am impressed how they can sell buggy devices like this. Got better significantly in last few versions of zwave-js but now I need at least scheduled soft reset to mitigate this issue which happens every few days. Ideally if I can make script that detects this state and soft-reset only when it happens.

**Reply by AlCalzone** [reply]
While they have their own firmwares, those are all based on Silicon Labs' official firmwares and the Z-Wave SDK. And being honest, the entire 7.x firmware line has been terrible.

Honestly I'm not sure how to detect that state from outside, but calling soft-reset is possible via the API. Interesting that this works for you. I tried it at some point and it didn't help for a lot of users.

Anyways, the next official firmware release is supposed to fix this issue, at least I was told so.

**Comment by AlCalzone** [comment]
Okay, I'm told 7.21.0 **probably** does contain the fix, but Silabs aren't sure enough to list it as fixed in the release notes. Please ask your stick manufacturer for a firmware based on SDK 7.21.0 and/or see https://github.com/kpine/zwave-js-server-docker/wiki/700-series-Controller-Firmware-Updates-(Linux)#firmware-downloads for links to "official" Silabs firmwares. Let me know if the problem persists on that firmware.

**Reply by fpytloun** [reply]
Great. I asked support and they confirmed firmware for Razberry with 7.21.0 SDK will be release this month.

**Reply by larstobi** [reply]
I have used 7.21.0 with my Z-Stick 7 since 9th January, and it is still not fixed. It seems I get the unable to transmit error message between 10 and 20 times per day.

**Reply by AlCalzone** [reply]
I think I posted this somewhere else, and forgot about this thread.
7.21.0 doesn't contain the fix. Several users are still having issues.

Silicon Labs ate aware but I have no idea when they'll fix it.

**Reply by iscofield** [reply]
This is absolutely asinine at this point that this is still an ongoing issue.  Does anyone have any clue as to whether they’re actually working to fix this issue or when a new firmware is coming?  This stick is about as useful as a rock in my backyard at this point.  I know this isn’t the fault of anyone here in this thread/issue, but seems like you all are more in the know at least.

**Reply by fpytloun** [reply]
Yes, unfortunately still no new firmware or fix. I am also very upset about it so I ordered Zooz 800 series (they have Raspberry HAT and USB dongle version) and will try to migrate to it as it might be more stable and migration by NVMe restore should be supported now.

**Reply by iscofield** [reply]
I am using the 800 series Zooz stick and am having these issues. So I would very much not direct you down that path. It seems like at this point 700 is a dumpster fire and so is 800, so the only safe solution is to drop down to 500.

**Reply by jasonkelly214** [reply]
Yeah my entire network has been dead in the water since like September. I have devices I haven't even bothered trying to pair just sitting waiting. Even more annoyingly, I have like 40 zooz sensors on doors, windows,etc that all need their batteries changed and I'm not even able to utilize them. 

This was replied to today on the SI forum: https://community.silabs.com/s/question/0D7Vm000000wAztKAE/detail?language=en_US&fromEmail=1&s1oid=00DA0000000L2kI&s1nid=0DB1M000000k9ha&s1uid=0058Y00000AQ9cI&s1ext=0&emkind=chatterCommentNotification&emtm=1710475676169

**Reply by iscofield** [reply]
If the mid-April date is to be believed, then that would be fantastic.

**Comment by maivorbim** [comment]
Can confirm this as well on a Zstick 7 running Aeotec beta firmware FW: v7.21.0. I am back to using a Zstick Gen5 which has vastly better performance currently than Zstick 7 due to buggy fw. Hopefully Silicon Labs fixes this sooner than later. You can follow here to see when a new FW drops: https://github.com/SiliconLabs/gecko_sdk/releases

**Comment by maivorbim** [comment]
Z-wave 7.21.1.0 was released https://github.com/SiliconLabs/gecko_sdk/releases hopefully the issue will be fixed once manufacturers provide updated FW

**Reply by AlCalzone** [reply]
It does not fix the issue - they still list it under "known issues" with no workarounds.

**Comment by Anakins** [comment]
Hello, same here (Jeedom + aeotec z stick 700).
First test : Downgrade do 18.8. Same Issue
Go back in 18.8 with another identical controller (new product). Same Issue.

I hope Aeotec will have a new firmware because it is very urgent now.
Problem since a couple of month now...

**Reply by AlCalzone** [reply]
Silicon labs have a fix in their next patch release which is scheduled for mid April.
Aeotec use that firmware unchanged, Zooz build their own based on the official one.

**Reply by Anakins** [reply]
Hello, 

Do you think if I change with a zooz controller (800 instead my aeotec 700), I will have the same issue ?

Thanks

**Reply by AlCalzone** [reply]
Yes. All firmwares are based on the one Silicon Labs develops. Zooz only change the manufacturer IDs so their controller gets detected as a Zooz device.

**Reply by fpytloun** [reply]
I just received Zooz 800 GPIO module and USB stick so I am going to try to migrate from RaZberry 7 Pro board to this one and let you know if it's better and more stable.

**Reply by fpytloun** [reply]
So today I migrated from RaZberry 7 Pro to Zooz 800 GPIO (on RaspberryPi 4) using NVM backup/restore and it went fine but bug is still present and overall experience is basically same as with 700 series.

**Comment by ioguy** [comment]
7.21.2 was just released.
It doesn't fix the lockup (still listed as a known issue in the release notes) and for anyone using BE469ZP locks, don't touch it.  Since upgrading I can't get them to talk at all.  Dead in the water.

**Reply by iscofield** [reply]
It's unreal how this continues to be a problem.  This 800 stick is essentially worthless.
Thank you for the heads up, as I have several of those locks :(

**Reply by fpytloun** [reply]
Yes both 800 and 700 has same issue. I have a little bit better stability after migrating to 800 than I had with 700 but it still happens and I also don't understand how is it possible that such severe issue, making these devices very unreliable can be present for so long.

**Reply by larstobi** [reply]
https://www.silabs.com/documents/public/release-notes/SRN14930-7.21.2.0.pdf

**Reply by fpytloun** [reply]
TL;DR

<img width="734" alt="image" src="https://github.com/zwave-js/node-zwave-js/assets/948799/52262514-4a0e-4781-924b-231a84625fde">

**Reply by ioguy** [reply]
FYI, the BE469ZPs are now working again.  It took about 12 hours of whatever mystery Z-wave chip settling was happening, but I can now re-interview and use the locks.

**Reply by jasonkelly214** [reply]
Wow. This is like the third time they've "found the problem" which will be released soon snd and the third time they actually haven't. My entire network dead since ~September.

**Comment by Hedda** [comment]
> 7.21.2 was just released.
> It doesn't fix the lockup (still listed as a known issue in the release notes) and for anyone using BE469ZP locks, don't touch it. Since upgrading I can't get them to talk at all. Dead in the water.

FYI, anyone can register in Silabs community and post feedback on known ID # 1227385 to Silicon Labs here:

https://community.silabs.com/s/question/0D5Vm0000036y3PKAQ/controller-reports-being-jammed-during-usage-and-provokes-the-zwave-network-to-stall-for-several-seconds

and

https://community.silabs.com/s/question/0D7Vm000000wAztKAE/detail?language=en_US&fromEmail=1&s1oid=00DA0000000L2kI&s1nid=0DB1M000000k9ha&s1uid=0058Y00000AQ9cI&s1ext=0&emkind=chatterCommentNotification&emtm=1710475676169

**Comment by austwhite** [comment]
IS this still being checked on?   Si seem to indicate 7.21.4 might help with the issue.....

**Reply by Hedda** [reply]
It is still and issue and still not fully fixed in 7.21.4 nor in 7.22 either;

https://community.silabs.com/s/question/0D7Vm000003Ox3eKAC/controller-reports-being-jammed-during-usage-and-provokes-the-zwave-network-to-stall-for-several-seconds?language=en_US&fromEmail=1&s1oid=00DA0000000L2kI&s1nid=0DB1M000000k9ha&s1uid=0051M00000Aiu7C&s1ext=0&emkind=chatterCommentNotification&emtm=1722965525870

”_**The 7.22 SDK contains improvements for the controller lockup issue, but as others pointed out, this applies to the 800 series devices only. The 700 series devices will be supported in the upcoming 7.21.x releases. The next patch release (7.21.4) could be expected in the middle of August and will contain improvements for the lockup.**_”
